trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/deploy_rules/action.py

```diff
@@ -12,20 +12,20 @@ class DeployRules(komand.Action):
 
     def __init__(self):
         super(self.__class__, self).__init__(
-                name='deploy_rules',
-                description=Component.DESCRIPTION,
-                input=DeployRulesInput(),
-                output=DeployRulesOutput())
+                name = 'deploy_rules',
```

Comment:
Arguments are the exception to the whitespace rule for keyword arguments, see this section of the PEP8:
```
Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter.
```
https://www.python.org/dev/peps/pep-0008/#id28

Reply:
yeah it felt weird when I did it. Makes sense.